### PR TITLE
Locale ru_RU. Update translation

### PR DESCRIPTION
**Pull request by Arttse**
Wow! Thanks for the reminder! Too much time has passed since last time. Ready to update.

### CODE REVIEW

File: youtube_dl_gui/locale/ru_RU/LC_MESSAGES/youtube_dl_gui.po [hunk 1]
```diff
@@ -72,71 +70,66 @@ msgstr "Добро пожаловать"
...
 #: youtube_dl_gui/mainframe.py:534
 msgid "Delete"
-msgstr ""
+msgstr "Удалить"
 
 #: youtube_dl_gui/mainframe.py:108
-#, fuzzy
 msgid "Play"
-msgstr "Плейлист"
+msgstr "Проиграть"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/ru_RU/LC_MESSAGES/youtube_dl_gui.po]: "Проиграть" translates to "lose", which does not much with the msgid (Play).
Are you sure this is correct?

Comment (Arttse) [inline youtube_dl_gui/locale/ru_RU/LC_MESSAGES/youtube_dl_gui.po]: Good catch! Word "Проиграть" has a different meaning for current context. In this case, it refers to the ["Проиграть видео"](https://translate.google.ru/#ru/en/%D0%9F%D1%80%D0%BE%D0%B8%D0%B3%D1%80%D0%B0%D1%82%D1%8C%20%D0%B2%D0%B8%D0%B4%D0%B5%D0%BE) or ["Проиграть аудио"](https://translate.google.ru/#ru/en/%D0%9F%D1%80%D0%BE%D0%B8%D0%B3%D1%80%D0%B0%D1%82%D1%8C%20%D0%B0%D1%83%D0%B4%D0%B8%D0%BE), which translates to "Play video" and "Play audio". Ok, I think for clarity I'll replace with another word "Воспроизвести", which means the same :+1:

**Comment by Arttse** [comment]
I made some edits, think everything is ok. And I think you can merge now.

**Comment by MrS0m30n3** [comment]
Great job @Arttse, thanks for your time.

**Pull request merged**